### PR TITLE
feature/grants-2916

### PR DESCRIPTION
**Pull request by yunusfirat**

### CODE REVIEW

File: app/config/question-bank.js [hunk 1]
```diff
@@ -1514,6 +1514,79 @@ const questionBank = {
...
+              heading: 'Eligibility',
+              content: [{
+                para: 'To be eligible for grant funding, your robotic technology must:',
+                items: ['have a sensing system and can understand their environment', 'make decisions and plan', 'can control its actuators (the devices that move robot joints)', 'work in a continuous loop'],
+                additionalPara: 'Automatic technology must fit at least 2 of these eligibility criteria. '
+              }]
+            }]
+          },
+          type: 'multi-input',
+          allFields: [
```
[the remark this answers is not in the document]
Comment (yunusfirat) [reply app/config/question-bank.js:1558]: We use single-answer for radio buttons I think the idea of using multi-input is using all field items. What's wrong if we do the way I have done here?

File: app/messaging/email/create-submission-msg.js [hunk 2]
```diff
@@ -204,14 +204,14 @@ function getEmailDetails (submission, desirabilityScore, rpaEmail, isAgentEmail
       projectImpact: submission.projectImpact ?? ' ',
       isDataAnalytics: submission.dataAnalytics ? 'Yes' : 'No',
       dataAnalytics: submission.dataAnalytics ?? ' ',
-      dataAnalyticsScore: submission.dataAnalytics && submission.projectSubject !== PROJECT_SUBJECT_SLURRY ? getQuestionScoreBand(desirabilityScore.desirability.questions, 'robotics-data-analytics') : ' ',
+      dataAnalyticsScore: submission.dataAnalytics && submission.projectSubject !== PROJECT_SUBJECT_SLURRY ? getQuestionScoreBand(desirabilityScore.desirability.questions, 'data-analytics') : ' ',
```
Comment (zavies97) [inline app/messaging/email/create-submission-msg.js:207]: can we revert the changes in this file for now?

File: app/messaging/create-msg.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ function getDesirabilityAnswers (request) {
   try {
     let val = {}
     const projectSubject = getYarValue(request, 'projectSubject')
-    if (projectSubject === 'Robotics and Innovation') {
+    if (projectSubject === 'Robotics and automatic technology') {
```
Comment (zavies97) [inline app/messaging/create-msg.js:28]: Can we revert the changes in this file for now pls? Updating scoring in another PR but it will take some time

File: test/unit/app/messaging/create-msg.test.js [hunk 4]
```diff
@@ -23,15 +23,14 @@ describe('create-msg', () => {
       energySource: ['value'],
       agriculturalSector: ['value'],
       dataAnalytics: 'testing',
-      technology: 'testing'
+      technologyUse: 'testing'
```
Comment (zavies97) [inline test/unit/app/messaging/create-msg.test.js:26]: This file can be reverted back to how it was after create-msg is reverted

**Review by zavies97 — APPROVED** [review]
